ui/raidboss/data/03-hw/trial/ravana-ex.js

```diff
@@ -5,4 +5,241 @@
   timelineFile: 'ravana-ex.txt',
   triggers: [
   ],
+  timelineReplace: [
+    {
+      'locale': 'de',
+      'replaceSync': {
+        'Dance to the song of ringing steel': 'Dance to the song of ringing steel', // FIXME
+        'Ravana readies The Seeing': 'Ravana readies The Seeing', // FIXME
+        'Ravana starts using Bloody Fuller on Ravana': 'Ravana starts using Bloody Fuller on Ravana', // FIXME
+        'Ravana starts using Final Liberation on Ravana': 'Ravana starts using Final Liberation on Ravana', // FIXME
```

Comment:
This is the same as the Alex stuff.  I would really rather not have replacement syncs that have full lines in them.

Reply:
This looks like it's still an issue.